src/ui/public/vislib/styles/_layout.less

```diff
@@ -19,6 +19,27 @@
   overflow: visible;
 }
 
+/* SVG Element Default Styling */
+.vis-wrapper {
```

Comment:
if i am not mistaken our style guide advises against this nesting ?

Reply:
@snide @cjcenizal Is this nesting of rules fine  or should this be rewritten?

Reply:
It's OK in the case of SVG elements since you can't really apply classes to them.

Reply:
Just make sure that `.vis-wrapper` is applied to nodes as close to the leaves of the DOM tree as possible. We want to avoid these selectors from applying to any SVGs which are unrelated to charts and things.